packages/smooth_app/lib/helpers/background_task_helper.dart

```diff
@@ -0,0 +1,181 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/utils/CountryHelper.dart';
+import 'package:smooth_app/database/dao_product.dart';
+import 'package:smooth_app/database/local_database.dart';
+import 'package:smooth_app/query/product_query.dart';
+import 'package:task_manager/task_manager.dart';
+
+const String IMAGE_UPLOAD_TASK = 'Image Upload';
+const String PRODUCT_EDIT_TASK = 'Product Edit';
+Future<TaskResult> callbackDispatcher(LocalDatabase localDatabase) async {
```

Comment:
As the name of the method is not straightforward, could you document a little bit?